app/controllers/food_types_controller.rb

```diff
@@ -10,10 +10,10 @@ def index
   end
 
   def destroy
-    @l = FoodType.find(params[:id])
-    authorize! :destroy, @l
-    @l.active = false
-    @l.save
+    @food_type = FoodType.find(params[:id])
+    authorize! :destroy, @food_type
+    @food_type.active = false
+    @food_type.save
```

Comment:
I think `@food_type` doesn't actually have to be an instance variable here. I'd be happy merging this as-is, but how would you feel about changing this to be a local var (removing `@`) as part of this PR?